issue_1/contracts/WordRegistry.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4;
+
+import "hardhat/console.sol";
```

Comment:
Please remove this before deploying, it uses deployment gas, and sometimes you might forget `console.log` that use transaction gas.

Reply:
ack